src/components/Hero.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import {Link} from "react-router-dom";
 import {MarketingButton} from "../styles/Button";
 import {Wrapper, PizzaPosition, TopHero, SubHero, TrustSection, QuoteSection, DetailsSection} from "../styles/Header";
-import {gucci, go, graphql, electron, next, sauceFull, bdougie} from "../images";
```

Comment:
@alotoftype can you also delete these files from the repo?

Reply:
yup will do